src/parser/deathknight/frost/CHANGELOG.tsx

```diff
@@ -1,11 +1,12 @@
 import React from 'react';
 
-import { Khazak } from 'CONTRIBUTORS';
+import { Khazak, LeoZhekov } from 'CONTRIBUTORS';
 import SPELLS from 'common/SPELLS';
 import SpellLink from 'common/SpellLink';
 import { change, date } from 'common/changelog';
 
 export default [
+    change(date(2020, 10, 22), 'Replaced deprecated StatisticBoxes with Statistic for frost DK', [LeoZhekov]),
```

Comment:
```suggestion
    change(date(2020, 10, 22), 'Replaced deprecated StatisticBoxes with Statistic', [LeoZhekov]),
```

This is a big nitpick, but I recently decided to remove the words Frost Death Knight from my changelogs because its redundant to say FDK when you're already in the FDK changelog.